src/player.cpp

```diff
@@ -1824,6 +1824,16 @@ bool player::is_immune_effect( const efftype_id &effect ) const
     return false;
 }
 
+bool player::drag_check() const
+{
+    if (get_str() + (get_per() / 3) + (get_dex() / 4) > rng(4, 15) || is_throw_immune()){
```

Comment:
Could add unarmed combat somewhere here. Also possible bonus for having a grab break technique.